CHANGELOG.md

```diff
@@ -19,6 +19,11 @@
 * Fixed a bug in `--experimental-reset-state-function` support for heap reset.
   [#4665](https://github.com/wasm-bindgen/wasm-bindgen/pull/4665)
 
+### Changed
+
+* Unconditionally use `TextEncoder`/`TextDecoder` for string encoding/decoding.
+  [#TODO]
```

Comment:
^

Reply:
Would be nice if there was some action that automatically fills in the PR number (since I don't know it at the time of commit)